imap_processing/mag/mag_cdf_attrs.py

```diff
@@ -42,6 +42,38 @@
     instrument_base=mag_base,
 )
 
+mag_l1a_norm_mago_attrs = GlobalDataLevelAttrs(
+    "L1A-raw-norm-mago>Level 1A MAGo normal rate",
```

Comment:
I recently found out that we need to have in this format, `data_type=<datalevel>_<descriptor>`. Can you change same thing in the below attrs as well?
```suggestion
    "L1A_raw-norm-mago>Level 1A MAGo normal rate",
```

Reply:
Yes, good catch, I should have updated this after your attr PR!